docs/modules/ROOT/pages/context-population.adoc

```diff
@@ -0,0 +1,50 @@
+= Context population
+
+The request context can be populated with a number of different values to influence the behaviour of the Neo4j GraphQL Library.
+
+== `driverConfig`
+
+== `executionContext`
+
+The Neo4j driver, session or transaction which will be used to execute the translated query.
+
+== `jwt`
+
+A decoded JWT payload A decoded JWT payload which can be provided for use in authentication and authorization.
+
+Takes precedence over `token` if provided.
+
+For example:
+
+[source, json, indent=0]
+----
+{
+    sub: "1234567890",
+    name: "John Doe",
+    iat: 1516239022,
+}
+----
+
+== `neo4jDatabaseInfo`
+
+== `queryOptions`
+
+== `token`
+
+The bearer token to be decoded/verified for use in authentication and authorization.
```

Comment:
Not so sure about this suggestion, though:

```suggestion
The Bearer `token` to be decoded/verified for use in authentication and authorization.
```

Reply:
Fairly sure "bearer" should be left lower-case. 🙂